src/plugins/kibana_react/public/page_template/solution_nav/solution_nav_collapse_button.scss

```diff
@@ -0,0 +1,34 @@
+.kbnPageTemplateSolutionNavCollapseButton {
+  // transition: left $euiAnimSpeedFast $euiAnimSlightResistance,
+  //             opacity $euiAnimSpeedFast $euiAnimSlightResistance;
+  position: fixed;
+  opacity: 0;
+  left: 240px - $euiSize;
+  z-index: 2;
+
+  @include euiCanAnimate {
+    transition: opacity $euiAnimSpeedFast, left $euiAnimSpeedFast;
+  }
+
+  &:hover,
+  &:focus {
+    transition-delay: 0s !important;
+  }
+
+  .kbnPageTemplate__pageSideBar:hover &,
+  &:hover,
+  &:focus {
+    opacity: 1;
+    left: 240px - $euiSizeL;
+  }
+
+  .kbnPageTemplate__pageSideBar:hover & {
+    transition-delay: 2s;
```

Comment:
Gave this a spin. I'd suggest making this a much lower value. Maybe `$euiAnimSpeedSlow` if you're worried about it being distracting.

Reply:
Yeah it's definitely finding the right balance. I def thought that all of our anim speeds though were too fast. I also am trying to figure out to get the button's focus/hover to be quicker but the overall nav:hover seems to take precedence. I also need to add in that maybe it comes in when the nav is in focus/focus-within.